master/buildbot/test/unit/test_reporter_gerrit.py

```diff
@@ -463,3 +464,22 @@ def testBuildGerritCommand(self):
             'ssh',
             ['ssh', 'user@serv', '-p', '29418', 'gerrit', 'review', '--project project',
              "--message 'bla'", '--verified 1', 'revision'])
+
+        # now test the notify argument
+        gsp.gerrit_notify = 'OWNER'
+        gsp.processVersion("2.6", lambda: None)
```

Comment:
do you mind clarifying when to use `'` instead of `"`? For one of the other comments above (in the .extend comment), you suggested I use `'` instead of `"` is why I ask.